crates/chain/src/lib.rs

```diff
@@ -97,5 +100,58 @@ pub mod collections {
     pub use hashbrown::hash_map;
 }
 
+/// Number of seconds that have elapsed since 00:00:00 UTC on 1 January 1970, the Unix epoch.
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, PartialEq, Eq, Copy, PartialOrd, Ord, core::hash::Hash)]
+pub struct UnixSeconds(u64);
```

Comment:
`bitcoin::absolute::Time` may be more appropriate.

Reply:
> `bitcoin::absolute::Time` may be more appropriate.


![image](https://github.com/bitcoindevkit/bdk/assets/133089376/54342de8-4fc7-43ed-8429-df4a59a6a457)

Reply:
Well clearly the default is ambiguous so if there's a sensible default in your struct you should implement it manually and if there isn't you shouldn't implement it.

Reply:
> `bitcoin::absolute::Time` may be more appropriate.

Good point, I didn't see `Time` and `Height` are both exposed under `bitcoin::absolute::`. Should certainly use those paths for `bdk_chain`.